src/components/ResourceExplorer/index.vue

```diff
@@ -1,51 +1,59 @@
 <template>
   <div>
-    <el-table ref="singleTable" :data="resources" v-loading="loading">
-      <el-table-column label="资源路径" min-width="100px">
-        <template slot-scope="scope">
-          {{ scope.row.path }}
-        </template>
+    <el-table ref="singleTable" :data="resources" highlight-current-row>
+      <el-table-column label="资源路径" min-width="80px">
+        <template slot-scope="scope">{{ scope.row.path }}</template>
       </el-table-column>
-      <el-table-column label="资源类型" width="110px">
-        <template slot-scope="scope">
-          {{ scope.row.stubType }}
-        </template>
+      <el-table-column label="资源类型" min-width="50px">
+        <template slot-scope="scope">{{ scope.row.stubType }}</template>
       </el-table-column>
-      <el-table-column label="属性" min-width="100px" show-overflow-tooltip>
+      <el-table-column label="属性" min-width="100px">
         <template slot-scope="scope">
-          <span>{{ JSON.stringify(scope.row.properties)  }}</span>

```

Comment:
`properties` is an object so it should be json.stringify, and add show-overflow-tooltip to avoid overflow showing.